bin/git-co-upstream-pr

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+pull_request_number=$1
+local_branch_name=$2
+
+if git remote -v | grep upstream; then
```

Comment:
I _think_ this will also print the word `upstream` to the screen, which you can avoid like so:

```suggestion
if git remote -v | grep -q upstream; then
```

Reply:
I tried with both and didn't see any difference. I guess it doesn't hurt adding it. Thank you!